dist/pages/login.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
```

Comment:
You have placed both the login.html and profile.html files inside a `dist` folder. Any specific reason for this? You may be better of leaving these HTML files in the root and leave other compiled CSS, JS and images inside the current `src` folder.

Reply:
I usually put them in the root or a pages folder on the root when there are a lot of pages, I must of just assumed in this case it might go in distribution folder. I will move them back into the root for the moment then.

Reply:
For things like JS files, typically you would use `src` unless you keep things in the root folder. The dist folder may end up being automatically generated/made when you compile an app/site. 

I see you've switched the structure a bit. This can be marked as resolved.

Reply:
I see, in the future I will leave my dist folder for compiled and generated content then. Good to know, thanks 👍